core/network/fabric-interop-cc/contracts/interop/membership.go

```diff
@@ -39,15 +38,12 @@ func validateMemberCertChains(membership *common.Membership) error {
 }
 
 // Validate 'identity.Attestation' object against a message byte array
+// Should be removed?: to be confirmed in PR
```

Comment:
This function isn't getting used, but the same pattern is used in `func validateAttestationsList`. So, if you want, you can refactor the latter to call this function.

Reply:
The reason I didn't used this function is because, `cert` is already parsed in `validateAttestationsList`, now this function takes string and parse the cert again which seemed redundant.
If we remove parsing from this function, then all that remains is decode base64 signature and call `validateSignature`.

So that's why wasn't sure which one is better. Do you prefer this (remove parsing of cert from this function and take parsed certificate as parameter instead) way, or current way is better?

Reply:
Why not remove `parseCert` from `validateAttestationsList` and retain it in this function instead? If you need the result of the parsing for some other purpose, you can add it to the return value of this function.

Reply:
Done.